tests/unit/test_client.py

```diff
@@ -0,0 +1,102 @@
+import os.path
+
+from botocore.exceptions import ClientError
+import pytest
+
+from moto import mock_s3
+from dotenv import load_dotenv
+
+import re
+from IIIFingest.client import Client
+
+load_dotenv()
```

Comment:
FYI pytest has a plugin that can do this automatically: [pytest-dotenv](https://pypi.org/project/pytest-dotenv/)